tokio/src/sync/mpsc/bounded.rs

```diff
@@ -523,6 +523,21 @@ impl<T> Sender<T> {
         enter_handle.block_on(self.send(value)).unwrap()
     }
 
+    /// Checks if `Receiver` is still alive.
```

Comment:
```suggestion
    /// Checks if the channel has been closed. This happens when the
    /// [`Receiver`] is dropped, or when the [`Receiver::close`] method is
    /// called.
    ///
    /// [`Receiver`]: struct@crate::sync::mpsc::Receiver
    /// [`Receiver::close`]: fn@crate::sync::mpsc::Receiver::close
```

Reply:
Committed this and applied same to unbounded channels